tests/protocols/gmpv8/test_create_target.py

```diff
@@ -17,14 +17,43 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import unittest
+import warnings
 
 from gvm.errors import RequiredArgument, InvalidArgument
+
 from gvm.protocols.gmpv8 import AliveTest
 
 from . import Gmpv8TestCase
 
 
 class GmpCreateTargetCommandTestCase(Gmpv8TestCase):
+    def test_create_target_with_ignoring_make_unique(self):
```

Comment:
In the future, you could also `assert` that indeed the `DeprecationWarning` is issued, not just *a* warning (see 
https://docs.python.org/3/library/warnings.html#testing-warnings).

But I think your current test is quite sufficient for the current code.

Reply:
Thanks a lot for the hint! I've update the PR.